shieldapi/__init__.py

```diff
@@ -0,0 +1,37 @@
+import logging
+import os
+
+# Configure logging
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger("shieldapi")
+
+
+def check_mandatory_env_vars():
+    """
+    Checks if all mandatory environment variables are set.
+    Skips the check if SKIP_ENV_CHECK is set.
+    Raises an EnvironmentError if any are missing.
+    """
+    # Allow skipping the check if SKIP_ENV_CHECK is set
+    if os.environ.get("SKIP_ENV_CHECK"):
+        logger.info(
+            "Skipping mandatory environment variable check due to SKIP_ENV_CHECK."
+        )
+        return
```

Comment:
I think an empty `return` is not a good practise. It should be the other way around that the following code is only executed if `SKIP_ENV_CHECK` is `False`.

Reply:
I've never heard that. Can you explain why it's not a good practice?

Reply:
https://pylint.pycqa.org/en/v2.13.9/messages/refactor/useless-return.html

According to the pylint rules it is considered as a useless statement. Pylint rules are guidelines maintained by the community for best practise and clean code.

Reply:
Tested